test/spec/spec.form.js

```diff
@@ -546,107 +546,139 @@ describe('Form Controller', function () {
             form.getNextStep(req, res).should.be.equal('/base/next-page');
         });
 
-        describe('forking journeys', function () {
-
-            it('returns the fork target if the condition config is met', function () {
-                req.form.values['example-radio'] = 'conditionMet';
-                form.options.forks = [{
-                    target: '/target-page',
-                    condition: {
-                        field: 'example-radio',
-                        value: 'conditionMet'
-                    }
-                }];
-                form.getNextStep(req, {}).should.contain('/target-page');
+        describe('with stubbed _getForkTarget', function () {
```

Comment:
please revise the descriptions in these `describe`'s and `it`'s, they don't describe the features.